modules/openapi-generator/src/main/resources/Java/libraries/webclient/api.mustache

```diff
@@ -37,23 +37,96 @@ public class {{classname}} {
     private ApiClient apiClient;
 
     public {{classname}}() {
-        this(new ApiClient());
```

Comment:
thanks for the PR

can we keep 4-space as the indention to make it consistent with the rest of the file?